lamindb/examples/cellxgene/_cellxgene.py

```diff
@@ -143,6 +169,11 @@ def get_cxg_schema(
 
     from lamindb.models import Feature, Schema, ULabel
 
+    # Attempt to find the Schema early as building the Schema is expensive when looped
```

Comment:
There's probably a way to reconstruct/find the schema by querying the component schemas but I think that this is kind of Occam's razor.